Functions.Templates/Templates/DaprTopicTrigger-PowerShell/run.ps1

```diff
@@ -0,0 +1,24 @@
+
+using namespace System
+using namespace Microsoft.Azure.WebJobs
+using namespace Microsoft.Extensions.Logging
+using namespace Microsoft.Azure.WebJobs.Extensions.Dapr
+using namespace Newtonsoft.Json.Linq
+
+# Sample Dapr Service Invocation Trigger
+# See https://aka.ms/azure-functions-dapr for more information about using this binding
+# 
+# These tasks should be completed prior to running :
+#      1. Install Dapr
+#      2. Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[4.*, 5.0.0)"
```

Comment:
I don't think you need this. The template would only be included in the preview bundle. Customers would not see this template unless they have bundle name in host.json set to "Microsoft.Azure.Functions.ExtensionBundle.Preview"